File: python/03_contains_char.py
"""
¿Contiene el carácter?
Fácil
10 pts
Strings

Enunciado
Devuelve true si el string contiene el carácter dado, false si no.

Ejemplo
contains_char("hola", "o")  # True
contains_char("hola", "z")  # False
Puedes usar print() para depurar. Los resultados aparecen en la Consola de salida, no en el navegador.
"""


def contains_char(text: str, char: str) -> bool:
    return char in text

File: typescript/03-contains-char.ts
// ¿Contiene el carácter?
// Fácil
// 10 pts
// Strings

// Enunciado
// Devuelve true si el string contiene el carácter dado, false si no.

// Ejemplo
// containsChar("hola", "o") → true
// containsChar("hola", "z") → false

function containsChar(text: string, char: string): boolean {
  return text.includes(char);
}

// No modificar: necesario para evaluar el resultado.
export { containsChar };
